trawa/main.py

```diff
@@ -77,17 +81,21 @@ def read_conf(self, name):
         except yaml.YAMLError as exc:
             self.log.critical("YAML error %s", exc)
             exit(1)
+        for d in self.conf['dirs']:
+            if not 'file_mask' in d:
+                d['file_mask'] = '*'
```

Comment:
I added this code to deal with older configs, but you can remove it if consider that is not needed.